vtpm/src/char.rs

```diff
@@ -0,0 +1,267 @@
+// Copyright © 2022, Microsoft Corporation
```

Comment:
The filename `char.rs` does not make sense to me.

Reply:
The original Author loosely structured this code around https://github.com/qemu/qemu/tree/master/backends/tpm and https://github.com/qemu/qemu/blob/master/include/chardev/char-fe.h.

I will move to SocketCharDev abstraction and drop charbackend references. How about socket_char.rs for the filename. This module will be used while using CUSE or domain socket interfaces to talk to swtpm.